{
    "checksum": "d7cc0ad7eefcb81d1430f55fed3ce901d4283c580d9f062f49a60ca51fe82587",
    "file": "qt-Android-Fix-exception-when-using-Qt-to-create-a-serv.patch",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-diff"
    },
    "number_of_duplicates": 1,
    "package": "ausweisapp2",
    "path": "ausweisapp2/1.22.2-1/libs/patches/qt-Android-Fix-exception-when-using-Qt-to-create-a-serv.patch",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/ausweisapp2/1.22.2-1/",
        "metric": {
            "size": 22820
        },
        "pts_link": "https://tracker.debian.org/pkg/ausweisapp2",
        "sloc": [
            [
                "cpp",
                87286
            ],
            [
                "python",
                1371
            ],
            [
                "xml",
                1044
            ],
            [
                "java",
                637
            ],
            [
                "sh",
                147
            ],
            [
                "makefile",
                5
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian/ausweisapp2",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/ausweisapp2/1.22.2-1/libs/patches/qt-Android-Fix-exception-when-using-Qt-to-create-a-serv.patch",
    "stat": {
        "perms": "rw-r--r--",
        "size": 1839,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.22.2-1"
}